install.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 # Check virtualenv instalation and run setup for application
 # set +u because virtualenv fails without option
-if ! [ -x "$(virtualenv --version)" ]; then
+if ! [ -z "$(command -v virtualenv)" ]; then
     if ! [ "$(python -m venv -h)" ]; then
```

Comment:
The scipt install.sh was not working in my computer. 
```shell
/usr/bin/python: No module named venv
Virtualenv is not installed
```

It worked after a little change in line 27
```shell
if ! [ "$(python3 -m venv -h)" ]; then
```